src/ComparedQueryable/ComparedQueryable.nuspec

```diff
@@ -2,7 +2,7 @@
 <package >
   <metadata>
     <id>Buildium.ComparedQueryable</id>
-    <version>0.1.2</version>
+    <version>0.1.3</version>

```

Comment:
Since this hasn't been tested in production yet, I figured it needed fast iteration, so I followed [semver guidance](http://semver.org/#doesnt-this-discourage-rapid-development-and-fast-iteration).